server/spec/requests/api/v1/mqtt_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'Api::V1::MqttController', type: :request do
+  describe 'POST #publish' do
+    it 'publishes light on to the MQTT broker' do
+      client = double(MQTT::Client)
+      allow(MQTT::Client).to receive(:connect).and_yield(client)
+
+      expect(client).to receive(:publish).with(
+        ENV['MQTT_TOPIC'],
+        ENV['MQTT_MESSAGE']
```

Comment:
Depending on the result of the discussion on removing mqtt message as an env variable, this will need to be changed

Reply:
Yesssir sounds good